rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/rascal/RascalLanguageServices.java

```diff
@@ -76,6 +77,7 @@
 
 public class RascalLanguageServices {
     private static final IValueFactory VF = IRascalValueFactory.getInstance();
+    private static final ErrorRecovery RECOVERY = new ErrorRecovery(IRascalValueFactory.getInstance());
```

Comment:
```suggestion
    private static final ErrorRecovery RECOVERY = new ErrorRecovery(VF);
```

Reply:
That will not work: `VF` is of type `IValueFactory` while we need a `IRascalValueFactory` here.